FirebaseFunctions/Sources/Callable+Codable.swift

```diff
@@ -15,6 +15,14 @@
 import FirebaseSharedSwift
 import Foundation
 
+@available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *)
+public enum StreamResponse<Message: Decodable, Result: Decodable>: Decodable {
+  /// The message yielded by the callable function.
+  case message(Message)
+  /// The final result returned by the callable function.
+  case result(Result)
+}
+
```

Comment:
```suggestion
```

Delete – This is already checked into stremable-functions branch: https://github.com/firebase/firebase-ios-sdk/blob/streamable-functions/FirebaseFunctions/Sources/Callable%2BCodable.swift